### PR TITLE
Added integration tests for the candidate_list module

### PR DESCRIPTION
**Pull request by nicolasbrossard**
Added integration tests for the candidate_list module

### CODE REVIEW

File: modules/candidate_list/test/candidate_listTest.php [hunk 1]
```diff
@@ -77,56 +90,212 @@ function testCandidateListPageLoads()
...
+            array(self::$_TST0001)
+        );
+    }
+    /**
+     * Performs various searches by DCCID (and DCCID only).
+     *
+     * @return void.
+     */
+    function testFilterByDccId()
+    {
```
Comment (xlecours) [inline modules/candidate_list/test/candidate_listTest.php]: // Stop here and mark this test as incomplete.
        $this->markTestIncomplete(
          'This test is using a React component'
        );

File: modules/candidate_list/test/candidate_listTest.php [hunk 2]
```diff
@@ -77,56 +90,212 @@ function testCandidateListPageLoads()
...
         );
         $this->assertEquals("select", $feedbackOptions->getTagName());
     }
+    /**
+     * Performs various searches by PSCID (and PSCID only).
+     *
+     * @return void.
+     */
+    function testFilterByPscid()
+    {
```
Comment (xlecours) [inline modules/candidate_list/test/candidate_listTest.php]: // Stop here and mark this test as incomplete.
$this->markTestIncomplete(
'This test is using a React component'
);

**Comment by driusan** [comment]
@nicolasbrossard Can you check why this is failing?

**Comment by nicolasbrossard** [comment]
Seems like React and Travis do not get along together....It works on my sandbox but I can't get it to work on Travis. Will have to mark it as skipped for now.

**Pull request merged**